src/dropin-model.js

```diff
@@ -98,6 +99,10 @@ DropinModel.prototype.confirmDropinReady = function () {
   this._setupComplete = true;
 };
 
+DropinModel.prototype.verifyCardReady = function () {
```

Comment:
Open to suggestions on the name of this method. Had trouble coming up with a good name. Maybe something like `toggleShouldWaitForVerifyCard` but that seems too complex?

Reply:
GH is acting wild for me, so posting again. Not sure if this is more Swift-y but typically we would name a method returning a `bool` either something like `canVerifyCard` or `isVerifyCardReady`.

Reply:
I think it depends on how we're using and calling the function. If we're calling it inside of an if statement, I think it makes sense to call it `canVerifyCard` or `isVerifyCardReady` since it's returning a boolean.

In this case, it's being called as part of the `_threeDSecure.verify` flow, so I like that original idea of calling it `toggleShouldWaitForVerifyCard` or something similar. `toggleWaitForVerification`?